pymc_marketing/mmm/__init__.py

```diff
@@ -1,6 +1,20 @@
 from pymc_marketing.mmm import base, delayed_saturated_mmm, preprocessing, validating
 from pymc_marketing.mmm.base import MMM, BaseMMM
-from pymc_marketing.mmm.delayed_saturated_mmm import DelayedSaturatedMMM
+from pymc_marketing.mmm.delayed_saturated_mmm import Philly
```

Comment:
What does Philly stand for?

Reply:
If it is a replacement for `DelayedSaturatedMMM` I think:

(1) We need a name that is closer to the concept of MMM
(2) Need to add a deprecation warning instead of just removing it.